cmd/testpeers.go

```diff
@@ -569,15 +570,41 @@ func peerPingLoadTest(ctx context.Context, conf *testPeersConfig, tcpNode host.H
 	return testRes
 }
 
-func natOpenTest(ctx context.Context, _ *testPeersConfig) testResult {
-	// TODO(kalo): implement real port check
-	select {
-	case <-ctx.Done():
-		return testResult{Verdict: testVerdictFail}
-	default:
-		return testResult{
-			Verdict: testVerdictFail,
-			Error:   errNotImplemented,
-		}
+func dialTCPIP(address string, timeout time.Duration) error {
+	conn, err := net.DialTimeout("tcp", address, timeout)
+	if err != nil {
+		return errors.Wrap(err, "net dial")
+	}
+	if conn == nil {
```

Comment:
I am pretty sure this is impossible case when err and conn are nil at the same time.